src/Symfony/Component/Form/Extension/Core/DataTransformer/NumberToLocalizedStringTransformer.php

```diff
@@ -121,7 +121,7 @@ public function transform($value)
         }
 
         // Convert fixed spaces to normal ones
-        $value = str_replace("\xc2\xa0", ' ', $value);
+        $value = str_replace(array("\xc2\xa0", "\xe2\x80\xaf"), ' ', $value);
```

Comment:
`e2 80 af` is a narrow no-break space. They seem to've started using it in the latest ICU.